docs/app/Examples/modules/Sticky/Types/StickyAdjacentContextExample.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react'
+import { Grid, Header, Image, Rail, Segment, Sticky } from 'semantic-ui-react'
+
+const Placeholder = () => (
+  <Image
+    style={{ marginTop: 14, marginBottom: 14 }}
+    className='wireframe paragraph image'
+    src='/assets/images/wireframe/paragraph.png'
+  />
+)
+
+class StickyAdjacentContextExample extends Component {
+  constructor(props) {
+    super(props)
+    this.state = { contextRef: null }
+  }
+
+  handleContextRef = contextRef => {
+    if (!this.state.contextRef) {
+      this.setState({ contextRef })
+    }
+  }
```

Comment:
@mariolamacchia I understand why you did this and I don't see a better way, too :cry: However, possible @levithomason knows it.